sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/FileAsyncClient.java

```diff
@@ -760,6 +764,68 @@ private void channelCleanUp(AsynchronousFileChannel channel) {
             .map(this::uploadResponse);
     }
 
+    /**
+     * Uploads a range of bytes from one file to another file.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * <p>Upload a number of bytes from a file at defined source and destination offsets </p>
+     *
+     * {@codesnippet com.azure.storage.file.fileAsyncClient.uploadRangeFromURL#long-long-long-uri}
+     *
+     * <p>For more information, see the
+     * // TODO: Fix link
+     * <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/put-range">Azure Docs</a>.</p>
+     *
+     * @param length Specifies the number of bytes being transmitted in the request body.
+     * @param destinationOffset Starting point of the upload range on the destination.
+     * @param sourceOffset Starting point of the upload range on the source.
+     * @param sourceURI Specifies the URL of the source file.
+     * @return The {@link FileUploadRangeFromURLInfo file upload range from url info}
+     */
+    public Mono<FileUploadRangeFromURLInfo> uploadRangeFromURL(long length, long destinationOffset, long sourceOffset,
+        URI sourceURI) {
+        return uploadRangeFromURLWithResponse(length, destinationOffset, sourceOffset, sourceURI)
+            .flatMap(FluxUtil::toMono);
+    }
+
+    /**
+     * Uploads a range of bytes from one file to another file.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * <p>Upload a number of bytes from a file at defined source and destination offsets </p>
+     *
+     * {@codesnippet com.azure.storage.file.fileAsyncClient.uploadRangeFromURLWithResponse#long-long-long-uri}
+     *
+     * <p>For more information, see the
+     * // TODO: Fix link
```

Comment:
Same as the other TODO, also need to format them in the style:

`// TODO (alias): comment`